### DISCUSSION TITLE
File upload issue

### DISCUSSION THREAD

**Discussion by Sujai11** [primefaces/community#1355] [PrimeReact]
So i am using primereact file uploader. And somehow it is accepting video files as well for android phones in mobile browser even though i have wrote what formats it should accept. It is working fine in web browser.. Could you tell me what might be the issue???

**Comment by melloware** [comment]
you didn't post any code of your FileUpload?   are you sure Android respect the `accept` value?  Have you tried it on an  iPhone to truly isolate it to mobile or whether its and Android only problem?

**Reply by Sujai11** [reply]
This is the code:
<FileUpload
  mode="basic"
  name="pancard"
  accept="image/png,image/jpeg,application/pdf,image/jpg"
  maxFileSize={8000000}
                        
 />
 
 In iphone its working fine. The issue is only with android devices.

**Reply by melloware** [reply]
ok then i feel like its an Android specific bug but i am not sure how to go about debugging it as I don't use an Android.
